resource-service/src/main/resources/application-k8s.yaml

```diff
@@ -24,8 +24,6 @@ spring:
 spring:
   profiles: k8s-config
 
-# Kubernetes discovery by DNS for the pod name
-# i.e. map from what-i-call-a-service -> name-of-that-service's-pod
```

Comment:
Is there a reason that the comment `# Kubernetes discovery via DNS to the service loadbalancer` was not added to this file?

Reply:
reverted and readded